docs/ox_inventory/Guides/creatingItems.md

```diff
@@ -239,28 +239,28 @@ Like with other items the item must first be registered.
 },
 ```
 
-When registered you can define the item as a container under the `Items.containers` table in `/modules/items/server.lua`
+When registered you can define the item as a container  in `/modules/items/containers.lua`
 The key for the container is the `name` you gave it when registering the item.
 You can also define the number of slots, the maximum weight, blacklist and whitelist items.
 
-* itemName: `table`
-  * size: `{number, number}`
-    * First index is number of slots the container will have.
-    * Second index is the maximum weight of the container.
-  * blacklist: `table`
-    * Key-value pairs of item name and true
-    * `['itemName'] = true`
-  * whitelist: `table`
-    * Key-value pairs of item name and true
-    * `['itemName'] = true`
+* itemName:
+  * slots: number
+  * The number represents the amount of slots
+  * maxWeight: number
+  * The number represents the maximum weight within the container
+  * blacklist:
+    * Supports single and multiple items
+    * `{ 'testburger`, 'testburger2' }
+  * whitelist:
+    * Supports single and multiple items
+    * `{ 'testburger`, 'testburger2' }
```

Comment:
You have some formatting issues here. Firstly the descriptions need to be indented further. Secondly, you used a back tick instead of a single quote on `testburger`. 
Check the output:
![image](https://user-images.githubusercontent.com/96954031/226213096-5d537e8f-ca11-428a-ba19-b3797f0162d2.png)

Reply:
Right you are, my bad, will correct it now :)

Reply:
Should be resolved